test-aria.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Not sure this file should be added to the repo.

Reply:
agreed. removed